### ISSUE TITLE
Use of deprecated endpoints

### ISSUE THREAD

**Issue by fsargent** [AgencyPMG/node-mediamath#3]
The v1 API endpoint has been deprecated for quite a while now.

We (MediaMath) recommend people use the official Node SDK: https://github.com/MediaMath/t1-node

If this repository is still maintained, I'd suggest archiving it or putting a disclaimer on.

If there are any features in here that are not supported by the official MediaMath NodeSDK, let us know and we'll be happy to build them in, or accept pull requests.

**Comment by ChrisAlvares** [comment]
Is the reporting API supported? That is the only one this library currently supports.

**Comment by fsargent** [comment]
No, I don't believe it is supported. I've created an issue to support that next.

https://api.mediamath.com/reporting/v1/std is currently supported, but 
https://api.mediamath.com/api/v1/ has been deprecated.

I'll keep this issue updated until we can support the reports API.
